src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/PaymentAccountServiceImpl.java

```diff
@@ -83,12 +82,13 @@ public void deleteUserAccountMaps(Organisation organisation) {
     }
 
 
+    @Transactional
     public void deletePaymentAccountsFromOrganisation(Organisation organisation) {

```

Comment:
@transactional annotation should not be added here. It should be above editPaymentAccountsByOrganisation

Reply:
Addressed here: https://github.com/hmcts/rd-professional-api/pull/731/commits/ab5e48e12c529ac0a7a065a4eb4a4bfaa1ca8c1f